packages/main/src/Card.hbs

```diff
@@ -1,13 +1,16 @@
 <div
 	class="{{classes.main}}"
 	dir="{{rtl}}"
+	role="region"
+	aria-labelledby="{{heading}}"
```

Comment:
Can you check if this should be aria-label not aria-labelledby, cause aria-label expects a string, and aria-labelledby expects an id to refer

Reply:
Should be aria-labelledby="{_id}}-heading"